lib/src/rows/Rows.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import { RowProps, RowsProps } from "./types";
+
+const DxcRows = ({
+  alignX = "stretch",
+  alignY = "stretch",
+  as = "div",
+  divider = false,
+  gutter = "0rem",
+  reverse = false,
+  children,
+}: RowsProps): JSX.Element => {
+  return (
+    <RowsContainer as={as} alignX={alignX} gutter={gutter} reverse={reverse}>
```

Comment:
I suggest calling this container just Rows to be more semantic.

Reply:
Done!